app/src/main/java/org/greenstand/android/TreeTracker/viewmodels/NewTreeViewModel.kt

```diff
@@ -61,23 +76,16 @@ class NewTreeViewModel(
         if (FeatureFlags.TREE_HEIGHT_FEATURE_ENABLED) {
             navigateToTreeHeight.postValue(newTree)
         } else {
-            saveTree(newTree)
+            createTreeUseCase.execute(newTree)
+            // Assign the current absolute step count to 'absoluteStepCountOnTreeCapture' to
+            // enable step count delta calculation for the next tree capture
+            stepCounter.absoluteStepCountOnTreeCapture = absoluteStepCount
+            stepCounter.disable()
             onTreeSaved.postValue(Unit)
             navigateBack.postValue(Unit)
         }
     }
 
-    private suspend fun saveTree(newTree: NewTree): Long {
```

Comment:
Unused methods, `Tree` in models package takes over the functionality that was supported by `NewTree`